### PR TITLE
Mise en place du Layout front + routes

### PR DESCRIPTION
**Pull request by MathieuCornut**

### CODE REVIEW

**Review by lescadev — CHANGES_REQUESTED** [review]
**CSS :**
- Please insert normalize CSS
- Rename file to style.css
- Correct indentation and code style
- REMOVE ALL FLOAT PROPERTIES !!!

**BASE LAYOUT :**
- Insert your CSS AFTER the stylesheets block
- The routes are not correctly used in navigation

**PAGE LAYOUT :**
- The title block is not correctly overridden
- One template for same pages types would be better

**CONTROLLER :**
- Rename to HomeController
- The routes are not correctly defined

**Review by VirginiePHILIPPE — APPROVED** [review]

File: src/Controller/HomeController.php [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    {
+        return $this->render('header/procurer.html.twig');
+        
+    }
+    
+    /**
+     * @Route("/", name="home")
+     */
+
+    public function home()
```
Comment (lescadev) [inline src/Controller/HomeController.php]: Please rename this action from "home" to "index"

File: src/Controller/HomeController.php [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    {
+        return $this->render('/home.html.twig');
```
Comment (lescadev) [inline src/Controller/HomeController.php]: Sure that the "/" is necessary ?

File: templates/base.html.twig [hunk 3]
```diff
@@ -3,52 +3,57 @@
...
     </head>
     <body>
         {% block navbar %}
-            <div class="tete">
+        
+        <header class="header">
                 <a href="/home"><img src="/logoloup.png" alt="logo lou pélou" class="logolou"></a>
                 <h1>Notre Monnaie nous appartient redonnons lui du sens ! </h1>
-                <a href="/header/login"><img src="/iconuser.svg" alt="connection/inscription"></a>
+                <a href="/header/login"><img src="/iconuser.svg" alt="connection/inscription" class="iconuser"></a>
```
Comment (lescadev) [inline templates/base.html.twig]: This is not a route call

File: templates/base.html.twig [hunk 4]
```diff
@@ -3,52 +3,57 @@
...
-
+        <link rel="stylesheet" href="/normalize.css">
+        <link rel="stylesheet" href="/style.css">
     </head>
     <body>
         {% block navbar %}
-            <div class="tete">
+        
+        <header class="header">
                 <a href="/home"><img src="/logoloup.png" alt="logo lou pélou" class="logolou"></a>
```
Comment (lescadev) [inline templates/base.html.twig]: This is not a route call

File: templates/base.html.twig [hunk 5]
```diff
@@ -3,52 +3,57 @@
...
+                    <a href={{ path('professionnels') }}>Professionnels</a>
                     </div>
                 </li>
-                <li><a href="/header/contacter">NOUS CONTACTER</a></li>
-                <li><a href="/header/evenement">EVENEMENTS</a></li>
-                <li><a href="/header/blog">BLOG</a></li>
+                <li><a href={{ path('contacter') }}>Nous contacter</a></li>
+                <li><a href={{ path('evenements') }}>Evenements</a></li>
+                <li><a href={{ path('blog') }}>Blog</a></li>
                 <li><a href="https://www.facebook.com/lechemindupelou/" target="_blank"><img class="fb" src="/Facebookicon.png" alt="fb"></a></li>
```
Comment (lescadev) [inline templates/base.html.twig]: No uppercase in filenames

File: templates/base.html.twig [hunk 6]
```diff
@@ -3,52 +3,57 @@
...
+        <footer class="footer">
             <ul>
-                <li><a href="/footer/mentionL">MENTIONS LEGALES</a></li>
-                <li><a href="/footer/cgv">CGV</a></li>
-                <li><a href="/footer/cgu">CGU</a></li>
-                <li><a href="/footer/statut">STATUT</a></li>
-                <li><a href="/footer/reglement">REGLEMENT INTERIEUR</a></li>
-                <li><a href="/footer/rendu">COMPTE RENDUAG 2019</a></li>
-                <li><a href="/header/contacter">CONTACT</a></li>
+                <li><a href={{ path('mentionL') }}>Mentions légales</a></li>
```
Comment (lescadev) [inline templates/base.html.twig]: No uppercase in routes name

**Review by lescadev — CHANGES_REQUESTED** [review]

File: src/Controller/HomeController.php [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+     * @Route("/procurer", name="procurer")
+     */
+    public function procurer()
+    {
+        return $this->render('header/procurer.html.twig');
+        
+    }
+    
+    /**
+     * @Route("/", name="index")
```
Comment (lescadev) [inline src/Controller/HomeController.php:19]: No need to rename this route, "home" was good

**Review by lescadev — APPROVED** [review]

**Review by Mihidari — APPROVED** [review]

**Review by NicolasROULON — APPROVED** [review]

**Pull request merged**